Firmware/tmc2130.cpp

```diff
@@ -272,17 +272,15 @@ void tmc2130_home_enter(uint8_t axes_mask)
 		tmc2130_wait_standstill_xy(1000);
 	for (uint8_t axis = X_AXIS; axis <= Z_AXIS; axis++) //X Y and Z axes
 	{
-		uint8_t mask = (X_AXIS_MASK << axis);
```

Comment:
**Note:** `(X_AXIS_MASK << axis)` is equal to `(1 << axis)` which is equal to `_BV(axis)`